README.md

```diff
@@ -118,6 +122,10 @@ Installation complete: enjoy time-tracking :-)
 
 ## Updating Kimai
 
+> **NOTE**
+> 
+> sudo -u www-data (a command which grants the "www-data" user temporary administrator/super-user privileges) is intended primarily for server applications. If you are installing Kimai 2 on a desktop computer - even for use in a local network, but where the computer primarily serves as a single user computer - you will avoid permission errors by subsituting "www-data" in the relevant commands with your username (i.e. sudo -u myusername ...).
```

Comment:
`sudo -u myusername` shouldn't be necessary, you could simply skip the sudo completely. Considering that you cloned via git with your own user, all files should already belong to it.
Maybe rephrase it to `subsituting "www-data" in the relevant commands with the username that runs the server`?

Reply:
Actually for updating I might be wrong, but for the first installation I am pretty sure.

And thats a big problem, normally these permission setups are so varying between installations that you won't be able to write something that works for everyone.